index.css

```diff
@@ -0,0 +1,105 @@
+* {
+  color: rgb(26, 26, 76) !important;
+  font-weight: 400;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-image: url(./static/images/background.png);
+  background-position: top 50px left 0;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav-bar {
+  display: flex;
+  padding: 5px 10px;
+  justify-content: space-between;
+}
+
+.nav-bar a {
+  font-size: 1.6rem;
+}
+
+#main-layout {
+  display: flex;
+  gap: 20px;
+  min-height: 100vh;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+}
+
+.flex-item {
+  width: 100%;
+  max-width: 600px;
+}
+
+.cont-header {
+  display: flex;
+  flex-direction: column;
+  line-height: 1.4;
+  margin: 0 15px;
+  padding-top: 70px;
+}
+
+.cont-header > span {
+  text-align: center;
+}
+
+.cont-header > span:first-child {
+  font-size: 1.83rem;
+}
+
+.cont-header > span:last-child {
+  font-size: 1.55rem;
+  color: rgb(37, 221, 114) !important;
+}
+
+.cont-desc {
+  padding: 10px 20px;
+  line-height: 1.5;
+  font-size: 1.075rem;
+  color: rgba(26, 26, 75, 0.7) !important;
+}
```

Comment:
- OPTIONAL kindly i can suggest you to add this values `font-size: 20px;
line-height: 32px;` to make your paragraph readable.

Reply:
Noted and appreciated